README.md

```diff
@@ -181,6 +194,25 @@ exporters:
     endpoint: "http://127.0.0.1:9411/api/v2/spans"
 ```
 
+### <a name="config-pipelines"></a>Pipelines
+Pipelines can be of two types:
+
+- metrics: collects and processes metrics data.
+- traces: collects and processes trace data.
+
+A pipeline consists of a set of receivers, processors, and exporters. Each receiver/processor/exporter must be specified
+in the configuration to be included in a pipeline and each receiver/processor/exporter can be used in more than one pipeline.
+
+The following is an example pipeline configuration. For more information, refer to [processor README.md](processor/README.md)
+```yaml
+pipelines:
+  traces/first:
```

Comment:
The `traces/first` key syntax is a) slightly unconventional b) has meaning that is not immediately obvious. It is worth explaining the "type/name" key convention somewhere close to this place.

Reply:
I'll remove '/first' and keep the example simple. This documentation will need a few more prs to make it a good landing page.